### DISCUSSION TITLE
% Value

### DISCUSSION THREAD

**Discussion by markus99** [nicufarmache/lovelace-big-slider-card#2] [Ideas]
Any thought to add a % value (either after name or as 2nd line below?)

Mushroom sliders aren't great, but what I'm using now and can show the value like this:

![image](https://user-images.githubusercontent.com/2835274/221998338-4f92fc01-b0bc-447f-9a6a-6268d285aaae.png)

**Comment by nicufarmache** [comment]
That's a good ideea. I'll try to implement it

**Comment by nicufarmache** [comment]
I added these new options in v1.0.8 
```yaml
show_percentage: true
bold_text: true
```
You can try it now
